algorithmia/manifest.json

```diff
@@ -0,0 +1,28 @@
+{
+  "display_name": "Algorithmia",
+  "maintainer": "support@algorithmia.io",
+  "manifest_version": "1.0.0",
+  "name": "algorithmia",
+  "metric_prefix": "algorithmia.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "Monitor metrics for machine learning models in production",
+  "guid": "bb9defff-03ae-4a22-93a7-6db498d37cd7",
+  "support": "contrib",
+  "supported_os": [
+    "linux"
+  ],
+  "public_title": "Datadog-Algorithmia Integration",
+  "categories": ["monitoring"],
+  "type": "check",
+  "is_public": false,
+  "integration_id": "algorithmia",
+  "assets": {
+    "dashboards": {},
```

Comment:
I see you've added a dashboard but it's not linked here. Did you use the `ddev meta dash export command` to properly add the dashboard to the integration? See more on the command + what it does here: https://datadoghq.dev/integrations-core/guidelines/dashboards/#exporting-a-dashboard-payload

Reply:
The link Christine provided offers some examples of best practices for dashboards. 

In particular, could you please add the following:

1. Algorithmia's Logo
2. A description of the dashboard to educate the user of its purpose/value (you can also add links to any Algorithmia documentation, a blogpost, Github, etc.)
3. Clear grouping titles, and graph titles, along with descriptions on the side explaining what the graph shows.

Similarly for the monitor, please ensure a thoughtful threshold value, add a description, and include an action playbook on how someone receiving the alert should handle it. 

Thanks!

Reply:
I added the dashboard and monitor to the manifest manually in 6e41962 since I was getting an error when using the `ddev meta dash export` command.

Reply:
I think that I followed all of the right steps, the error that I am getting is:

```
Traceback (most recent call last):
  File "/Users/koverholt/miniconda3/lib/python3.7/site-packages/toml/decoder.py", line 511, in loads
    multibackslash)
  File "/Users/koverholt/miniconda3/lib/python3.7/site-packages/toml/decoder.py", line 736, in load_line
    raise ValueError("Invalid date or number")
ValueError: Invalid date or number

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/koverholt/miniconda3/bin/ddev", line 8, in <module>
    sys.exit(ddev())
  File "/Users/koverholt/miniconda3/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/Users/koverholt/miniconda3/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/Users/koverholt/miniconda3/lib/python3.7/site-packages/click/core.py", line 1256, in invoke
    Command.invoke(self, ctx)
  File "/Users/koverholt/miniconda3/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/koverholt/miniconda3/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/Users/koverholt/miniconda3/lib/python3.7/site-packages/click/decorators.py", line 21, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/Users/koverholt/miniconda3/lib/python3.7/site-packages/datadog_checks/dev/tooling/cli.py", line 43, in ddev
    config = load_config()
  File "/Users/koverholt/miniconda3/lib/python3.7/site-packages/datadog_checks/dev/tooling/config.py", line 79, in load_config
    config.update(toml.loads(read_config_file()))
  File "/Users/koverholt/miniconda3/lib/python3.7/site-packages/toml/decoder.py", line 513, in loads
    raise TomlDecodeError(str(err), original, pos)
toml.decoder.TomlDecodeError: Invalid date or number (line 38 column 1 char 562)
```

Let me know if I missed anything by manually adding the dashboard and monitor.

Reply:
Also, thanks for the tips, @alexvetras. Let me get my `ddev` back in working order and I will add the changes that you mentioned regarding the dashboard and monitor best practices.

Reply:
I updated the dashboard and monitor in dc38a9f and 1fbdc3d to follow the documented best practices.

<img width="1511" alt="Screen Shot 2020-10-22 at 10 58 45 PM" src="https://user-images.githubusercontent.com/303236/96954989-345e4d00-14ba-11eb-98b0-0fe89b3c7cf7.png">

* Use screenboard instead of timeboard
* Insert logo and descriptive notes
* Add explanations of default operational metrics and how to add user-specified metrics
* Added monitor threshold, description, `recommended_monitor_metadata`, and steps to take upon an alert

Thanks again for the guidance here!